src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/BookingTest.kt

```diff
@@ -1569,6 +1569,8 @@ class BookingTest : IntegrationTestBase() {
 
   @Test
   fun `Create Confirmation on Booking returns OK with correct body`() {
```

Comment:
I was expecting to see the familiar set up for the parameterised test here:

```kt
  @ParameterizedTest
  @EnumSource(value = UserRole::class, names = [ "MANAGER", "MATCHER" ])
```

but perhaps the user with those roles was already defined?

Reply:
This is because the test is against a TA booking so works without roles - I've amended the test name to make this clear and added another one for Approved Premises that has the Parameterized setup